### PR TITLE
Missed some providers

### PR DESCRIPTION
**Pull request by awels**
causing havoc when running tests

Signed-off-by: Alexander Wels <awels@redhat.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/prometheus_test.go [hunk 1]
```diff
@@ -283,14 +283,22 @@ func getPrometheusSaToken(k8sClient *kubernetes.Clientset) (string, error) {
...
 		}
 	}
-	secret, err := k8sClient.CoreV1().Secrets(monitoringNs).Get(context.TODO(), secretName, metav1.GetOptions{})
-	if err != nil {
-		return "", err
-	}
-	if _, ok := secret.Data["token"]; !ok {
-		return "", fmt.Errorf("No token data in secret")
+	if secretName == "" {
+		// Since 1.24 SAs don't have tokens automatically generated, we determined the SA has no secret, so we
```
Comment (akalenyu) [inline tests/prometheus_test.go:287]: if you want to use the API directly you can check out what we did in CDI
https://github.com/kubevirt/containerized-data-importer/pull/2328/commits/79c8c331eb59195ffad6a30f30394e9807162833

File: cluster-sync/sync.sh [hunk 2]
```diff
@@ -43,8 +43,6 @@ metadata:
   name: hostpath-provisioner
 spec:
   imagePullPolicy: Always
-  imageRegistry: quay.io/kubevirt
```
Comment (akalenyu) [inline cluster-sync/sync.sh:46]: so we don't need this anymore? how come?

Comment (awels) [reply cluster-sync/sync.sh:46]: The new format of the CRD doesn't have those fields at all. So the webhook is rejecting this completely.

File: tests/prometheus_test.go (hunk 1 above)
Comment (awels) [reply tests/prometheus_test.go:287]: Thanks, I will fix that, looks much better than me calling kubectl.

File: cluster-sync/sync.sh (hunk 2 above)
Comment (awels) [reply cluster-sync/sync.sh:46]: I suspect that because I updated k8s to a newer version something is enforcing it now where it wasn't before.

Comment (akalenyu) [reply cluster-sync/sync.sh:46]: oh right yeah, seen that somewhere else too, should really dig out which k8s feature this is

**Comment by akalenyu** [comment]
/lgtm

second try

**Pull request merged**